WordPress/wordpress.org
# Update event list UI: Reorder event information and display event type

See https://github.com/WordPress/wordcamp.org/issues/894

This PR aims to improve the event list user interface by rearranging the information for each event and displaying the event type prominently. The changes include:

This update is being made in order to support the [The Next Generation of WordCamps](https://make.wordpress.org/community/2023/05/08/the-next-generation-of-wordcamps/) pilot event program. More traffic will be directed to this page (`community/events`) and clearly communicating the event type has been identified as a needed improvement.

- Moving the time information to the first line.
- Keeping the event name in the second line.
- Placing the location information in the third line.
- Introducing the event type and putting it before the location for better identification.

## Screenshots

| Before | After1 | After2 |
|   --       | --       |        -- |
| ![image](https://github.com/WordPress/wordpress.org/assets/18050944/7ab80074-d0cd-4113-aee1-96c160c28638) | ![Screen Shot 2023-06-13 at 2 36 33 AM](https://github.com/WordPress/wordpress.org/assets/18050944/70851070-f8fb-4fe4-b753-7d99d2848039) | ![Screen Shot 2023-06-13 at 2 36 20 AM](https://github.com/WordPress/wordpress.org/assets/18050944/6b587121-06c2-4de3-98ce-dcd62c061695) |

## Note

1. `mb_convert_case` was used because even though we don't seem to translate this page into any other languages, I guess we will in the future and this function will fit the case while `ucwords` won't. Also, I assume `type` in the database is specified as `UTF-8`.

2. ~I'm not sure if it's worth doing (or best to do) more on our current layout. Perhaps adding some labels before each piece of information, like `Time: 2:00 p.m.`, or adding a table for it, could be beneficial? What do you all think? And Is now a good time to bring the meta design into this discussion, or someone else should be tagged instead?~ ✅ 

## Testing Steps

Go to https://make.wordpress.org/community/events/ while sandboxed and check if the layout is the same as the screenshots above.
